packages/office-ui-fabric-react/src/components/Rating/Rating.scss

```diff
@@ -41,13 +45,28 @@ $Rating-largeIconSize: 20px;
 
     color: $Rating-defaultColor;
 
+    .halfStarContainer {
+      position:relative;
+    }
+
+    .halfStar{
+      overflow: hidden;
+      position:absolute;
+      left:0px;
+      top: 0px;
+    }
+
     &.starIsSelected {
       color: $Rating-selectedColor;
     }
 
     &.starIsDisabled {
       color: $Rating-disabledColor;
 
+      .halfStarContainer {
```

Comment:
This is because we have redefined the css again for half star element. Reason below